gradle/libs.versions.toml

```diff
@@ -0,0 +1,64 @@
+[versions]
+assertj = "3.27.3"
+aws = "2.30.3" # this is in mapping with iceberg repo.
+checkstyle = "10.21.3"
+errorprone = "2.36.0"
+errorproneSlf4j = "0.1.28"
+googleJavaFormat = "1.25.2"
+guava = "33.4.0-jre"
+hadoop = "2.7.3" # this is in mapping with iceberg repo.
+hive = "2.3.9" # this is in mapping with iceberg repo.
+iceberg = "1.7.1"
+immutables = "2.10.1"
+jacoco = "0.8.12"
+jandex = "3.2.3"
+junit = "5.12.0"
+logback = "1.5.17"
+logcaptor = "2.10.1"
+nessie = "0.101.3"
+nessieRunner = "0.32.2"
+picocli = "4.7.6"
+shadowPlugin = "8.1.1"
+slf4j = "1.7.36"
+
+[libraries]
+assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
+aws-sdk-apache-client = { module = "software.amazon.awssdk:apache-client", version.ref = "aws" }
```

Comment:
We should probably mention these in LICENSE, WDYT?

Reply:
> runtimeOnly(libs.aws.sdk.apache.client)

So, user has to supplement these jars. We are not shipping it. 

That being said. We do need to have LICENSE and NOTICE for the runtime jar (CLI) that we are shipping. Maybe in a follow up PR.

Reply:
Handling binary LICENSE later SGTM :+1:

Reply:
added in the PR description of TODO to track it.